presto-native-execution/presto_cpp/main/operators/LocalPersistentShuffle.h

```diff
@@ -21,21 +21,34 @@
 
 namespace facebook::presto::operators {
 
-/// LocalShuffleInfo can be used for both READ and WRITE.
-/// This struct is a 1:1 strict API mapping to
+/// LocalShuffleReadInfo and LocalShuffleReadInfo can be used for both READ and
```

Comment:
typo? XxxReadInfo and XxxReadInfo

Should this say something like "LocalShuffleWriteInfo is used for WRITE."